metadata-protobuf/proto/Metaprotocol.proto

```diff
@@ -156,11 +134,46 @@ message AppMetadata {
 
 message CreateApp {
     required string name = 1;
-    optional AppMetadata app_metadata = 3;
+    optional AppMetadata app_metadata = 2;
 }
 message UpdateApp {
     required string app_id = 1;
-    optional AppMetadata app_metadata = 3;
+    optional AppMetadata app_metadata = 2;
+}
+message DeleteApp {
+    required string app_id = 1;
+}
+
+message LeadRemarked {
```

Comment:
One more cosmetic change crossed my mind: I think this message should be named `ContentLeadRemarked`, since it is not a generic lead remark, but only a lead remark from a specific working group in this case.